NuGet.Config

```diff
@@ -3,20 +3,7 @@
   <packageSources>
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
     <add key="Local" value="tools/LocalNugetFeed" />
-    <add key="roslyn" value="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
     <add key="azure-sdk-tools" value="https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json" />
     <add key="azure-sdk-for-net" value="https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-net/index.json" />
   </packageSources>
-  <config>
-		<add key="globalPackagesFolder" value="restoredPackages" />
-    <add key="RestorePackagesPath" value="restoredPackages" />
```

Comment:
With this removed where will packages no restore to?

Reply:
Yes that is the default location.